src/components/TextListItem.tsx

```diff
@@ -30,91 +30,120 @@ export type ItemData = {
-      </>
-    );
-  }
+} = memo<Props>(
+  ({
+    containerStyle,
+    imageContainerStyle,
+    imageStyle,
+    item,
+    leftImage,
+    navigation,
+    noSubtitle,
+    showOpenStatus,
+    titleFirst
+  }) => {
+    const {
+      badge,
+      bottomDivider,
+      leftIcon,
+      onPress,
+      params,
+      picture,
+      routeName: name,
+      statustitle,
+      statustitleIcon,
+      subtitle,
+      teaserTitle,
+      title,
+      topDivider
+    } = item;
+    const navigate = () => navigation && navigation.push(name, params);
+    let titleText = <BoldText>{trimNewLines(title)}</BoldText>;
+
+    const textDirection: 'column' | 'column-reverse' = titleFirst ? 'column-reverse' : 'column';
+
+    let status = '';
+    if (showOpenStatus) {
+      const openStatus = isOpen(item?.params?.details?.openingHours);
+      status = openStatus?.open ? 'Jetzt geöffnet' : 'Geschlossen';
+    }
+
+    if (teaserTitle) {
+      titleText = (
+        <>
+          {titleText}
+          <RegularText small>{teaserTitle}</RegularText>
+        </>
+      );
+    }
 
-  if (statustitle) {
-    titleText = (
-      <>
-        {titleText}
-        <WrapperRow style={styles.statustitleWrapper}>
-          {!!statustitleIcon && statustitleIcon}
-          <RegularText small placeholder>
-            {statustitle}
-          </RegularText>
-        </WrapperRow>
-      </>
+    if (statustitle) {
+      titleText = (
+        <>
+          {titleText}
+          <WrapperRow style={styles.statustitleWrapper}>
+            {!!statustitleIcon && statustitleIcon}
+            <RegularText small placeholder>
+              {statustitle}
+            </RegularText>
+          </WrapperRow>
+        </>
+      );
+    }
+
+    // `title` is the first line and `subtitle` the second line, so `title` is used with our subtitle
+    // content and `subtitle` is used with the main title
+    return (
+      <ListItem
+        bottomDivider={bottomDivider !== undefined ? bottomDivider : true}
+        topDivider={topDivider !== undefined ? topDivider : false}
+        containerStyle={[styles.container, containerStyle && containerStyle]}
+        badge={badge}
+        onPress={() => (onPress ? onPress(navigation) : navigate())}
+        disabled={!navigation}
+        delayPressIn={0}
+        Component={Touchable}
+        accessibilityLabel={`(${title}) ${consts.a11yLabel.button}`}
+      >
+        {leftIcon ||
+          (leftImage && !!picture?.url ? (
+            <Image
+              source={{ uri: picture.url }}
+              style={[styles.smallImage, imageStyle && imageStyle]}
+              containerStyle={[
+                styles.smallImageContainer,
+                imageContainerStyle && imageContainerStyle
+              ]}
+            />
+          ) : undefined)}
+
+        <ListItem.Content style={{ flexDirection: textDirection }}>
+          {showOpenStatus && !!status && <RegularText small>{status}</RegularText>}
```

Comment:
`status` is not set if `showOpenStatus` is false, so checking `status` is enough here